src/main/java/io/vertx/core/http/WebSocketBase.java

```diff
@@ -175,6 +201,19 @@
   WebSocketBase binaryMessageHandler(@Nullable Handler<Buffer> handler);
 
   /**
+   * Set a pong message handler on the connection.  This handler will be invoked every time a pong message is received
+   * on the server, and can be used by both clients and servers since the RFC 6455 Sections 5.5.2 and 5.5.3 do not
+   * specify whether the client or server sends a ping.
+   *
+   * There is no ping handler since pings should immediately be responded to with a pong with identical content
+   *
+   * @param handler the handler
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  WebSocketBase pongHandler(@Nullable Handler<Buffer> handler);
```

Comment:
add a note saying that the pong handler will receive pong frames for sollicited ping but also unsolicited pong.